### PR TITLE
A login server so we can have something for our login screen to call

### PR DESCRIPTION
**Pull request by goofballLogic**

### CODE REVIEW

File: fake-server/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+const app = express();
+const port = 3000
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
+app.post('/', (req, res) => {
+    console.log(req.body);
+    const { username, password } = req.body || {};
+    if (username !== "james") {
```
Comment (dudeyds) [inline fake-server/index.js:12]: Please perform validation for password too, that way I know both username and password are correct in the POST request I send to the server

**Comment by dudeyds** [comment]
Please also merge from master to get README file, then explain how to run the server on a local device

File: fake-server/index.js (hunk 1 above)
Comment (goofballLogic) [reply fake-server/index.js]: ok i switched the logic order around and added password in

**Comment by goofballLogic** [comment]
done!

**Pull request merged**